### PR TITLE
update the learning outcomes for markup module

### PR DESCRIPTION
**Pull request by ivan-unfolds**

### CODE REVIEW

File: src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 ## Design
 
-- [ ] **Ensure our website displays well on screens of different sizes**
-- [ ] Use CSS media queries to ensure our content is always presented effectively
-- [ ] Demonstrate a mobile-first approach to designing a website with a great user experience
-- [ ] **Create an attractive and accessible colour palette for our project**
+- [ ] Use CSS media queries to ensure our content is always presented effectively on screens of different sizes
+- [ ] Demonstrate a mobile-first approach to building a website
 - [ ] Use CSS variables to apply repeated colours to HTML elements
+- [ ] Use CSS Flex to style children in a single-direction layout (ie a row or a column)
```
Comment (Albadylic) [inline src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md:13]:
```suggestion
- [ ] Use CSS Flexbox to style children in a single-direction layout (ie a row or a column)
```

File: src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+- [ ] Use CSS Grid to style children in two-direction layout
```
Comment (Albadylic) [inline src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md:14]: 3 Dimensional? i.e. Rows and Columns

Are we asking them to use both Flexbox and Grid; or either of them?

File: src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md [hunk 3]
```diff
@@ -2,14 +2,21 @@
...
+
+## Workflow
+
+- [ ] Ensure our git commit history tells a coherent story
```
Comment (Albadylic) [inline src/course/syllabus/pre-apprenticeship/markup/learning-outcomes.md:18]:
```suggestion
- [ ] Ensure our Git commit history tells a coherent story
```

**Review by Albadylic — APPROVED** [review]
Left a couple of suggestions here – looks good!

**Comment by sofer** [comment]
Merging now.

**Pull request merged**